### ISSUE TITLE
The Total amount is displaying the wrong amount

### ISSUE THREAD

**Issue by GriffGreen** [geleeroyale/donation-leaderboard#47]
the total amount raised in the wallet doesnt match the amount listed on the front page: 

<img width="457" alt="screen shot 2018-03-12 at 10 57 43 am" src="https://user-images.githubusercontent.com/14945613/37277176-3e02835c-25e4-11e8-9173-53af37f6e5ae.png">
it says 14.6 but etherscan says 14.28... https://etherscan.io/address/0x9cb8921aa376219950ba134c15d8f5ee2769c599

**Comment by wildmolasses** [comment]
I've got a fix for this that I can file a PR for. However, my fix calculates the total at 14.09, which is also the answer I get when I sum the external transaction values in R. Yet, there are 2 "internal transactions" on etherscan that seem to be impacting the etherscan total (@ 14.29).

So, I'll make a PR for the 14.09, and would be happy to discuss the internal tx stuff with you guys.

**Comment by geleeroyale** [comment]
Ok - I understand now ... interesting problem. maybe @sponnet has a clue whats going on here? In any case I merged your intermediate PR, since displaying less is in any case better than displaying more :-D

**Comment by GriffGreen** [comment]
I think we need to be listening for events... or at least thats what i hear people usu do... thats probably when you are directly using the chain, not just using etherscan's API...

**Comment by GriffGreen** [comment]
5 Gov points for the Fix though @emazurek :-D

**Comment by sponnet** [comment]
We can use the api to query the actual balance too, either via de Ether
scan api or via infura over web3... Let me know if you have a preference,
it's a small fix Imo

**Comment by geleeroyale** [comment]
I actually **wanted** (not anymore) to go the `query balance via etherscan` route instead of adding up values ourselves. Infura sounds interesting, but as long as we cannot handle everything via a node based backend, I would just use etherscan as well for this query.

A fundamental and not talked about problem is how to handle if funds are taken out of the monitored address, they would of course remain on the leaderboard, but the balance will be incorrect. It is good to inform donors somehow that funds haven been taken out.

griff means to listen for `donate` events, because we changed the event submitted from transactions in the multisig to `donate` - **would this work?** - see [https://github.com/Giveth/ScalingNowMultiSigWallet](https://github.com/Giveth/ScalingNowMultiSigWallet) for the multisig code (we actually deployed 1.1, not 1.2)
